Bag of words/Test_3.py

```diff
@@ -1,22 +1,45 @@
-import time
-import json
-import tornado
+
+
+import tornado  
+
+import os
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.metrics.pairwise import cosine_similarity
 import numpy as np
-
 vectorizer=TfidfVectorizer()
-text=open('newfile.txt')
-corpus = text.readlines()
-text.close()
+file_list = [f for f in os.listdir('.') if os.path.isfile(os.path.join('.', f)) and f.endswith("csv")]
```

Comment:
CSVs will be kept in a specific folder, use this within a function that takes the path of that directory and returns the list of file paths.